packages/protocol/contracts/libs/LibSharedConfig.sol

```diff
@@ -8,42 +8,57 @@ pragma solidity ^0.8.9;
 
 import "../L1/TaikoData.sol";
 
+/*
+> cd taiko-mono/packages/protocol/utils/generate_config
+> python3 main.py
+Expected block time (seconds): 20
+Expected proof time (minutes): 10
+Slot availability multiplier: 20
+Number of ZKPs required per block before verificaiton: 1
+Extra slots (e.g, 50 means 50% more slots): 100
+---------
+min num slots: 30
+---------
+initialUncleDelay: 10 minutes
+maxNumBlocks: 61
+zkProofsPerBlock: 1
+slotSmoothingFactor: 16789
+*/
+
 library LibSharedConfig {
     /// Returns shared configs for both TaikoL1 and TaikoL2 for production.
     function getConfig() internal pure returns (TaikoData.Config memory) {
         return
             TaikoData.Config({
                 chainId: 167,
-                maxNumBlocks: 2049, // up to 2048 pending blocks
-                blockHashHistory: 100000,
-                // This number is calculated from maxNumBlocks to make
-                // the 'the maximum value of the multiplier' close to 20.0
-                zkProofsPerBlock: 1,
-                maxVerificationsPerTx: 20,
-                commitConfirmations: 0,
-                maxProofsPerForkChoice: 5,
-                blockMaxGasLimit: 5000000, // TODO
-                maxTransactionsPerBlock: 20, // TODO
-                maxBytesPerTxList: 10240, // TODO
-                minTxGasLimit: 21000, // TODO
-                anchorTxGasLimit: 250000,
-                feePremiumLamda: 590,
-                rewardBurnBips: 100, // 100 basis points or 1%
-                proposerDepositPctg: 25, // 25%
+                maxNumBlocks: 61, // owner:daniel
+                blockHashHistory: 40, // owner:daniel
+                zkProofsPerBlock: 1, // owner:daniel
+                maxVerificationsPerTx: 20, //owner:david - TODO: what's the actual tx gas cost?
+                commitConfirmations: 0, // owner:daniel
+                maxProofsPerForkChoice: 3, // owner:daniel
```

Comment:
not sure will we also increase this value? seems it means right now we only allow 3 fastest honest provers?

Reply:
Only 3 for now.